code/modules/antagonists/blob/blobstrains/regenerative_materia.dm

```diff
@@ -7,7 +7,7 @@
 	complementary_color = "#AF7B8D"
 	message_living = ", and you feel <i>alive</i>"
 	reagent = /datum/reagent/blob/regenerative_materia
-	core_regen_bonus = 20
+	core_regen_bonus = 18
```

Comment:
this was 20 before the linked pr was made, but it did change point_rate_bonus from 3 to 1 so change that back